.github/scripts/agentic-supervisor.sh

```
@@ -162,15 +162,18 @@ has_jules_invoked_label() {
 # Helper: Collect all CodeRabbit inline comments on this PR
 collect_coderabbit_comments() {
   if [[ -n "$GH_TOKEN" && -n "$NUMBER" ]]; then
-    # Fetch raw comments JSON
-    COMMENTS_JSON=$(gh api "/repos/${GITHUB_REPOSITORY}/pulls/${NUMBER}/comments" 2>/dev/null || echo "[]")
+    # Create temp file for comments to avoid argument list too long errors
+    local comments_file=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)